bin/cij_fetch

```diff
@@ -82,12 +85,14 @@ def meta_reservoir(url):
 
     return None
 
+
 def meta_is_equiv(one, other):
     """Return True two different metas has the same size and md5sum."""
 
     return (one is not None and other is not None) and \
-            (one["length"] == other["length"]) and \
-            (one["md5"] == other["md5"])
+        (one["length"] == other["length"]) and \
+        (one["md5"] == other["md5"])
+
 
```

Comment:
To long expressions like this one, I usually add a parenthesis around the whole thing. It allows one to skip the line continuations, e.g.

```
return (
  (one is not None and other is not None) and
  (one["length"] == other["length"]) and
  (one["md5"] == other["md5"])
)
```

Reply:
Much nicer, thanks.